src/comp.rs

```diff
@@ -1,11 +1,13 @@
+use std::sync::Arc;
+
 use anyhow::Result;
 use itertools::Itertools;
 use phylotree::tree::{Comparison, Tree};
 use serde::Serialize;
 
 #[derive(Serialize, Default, Debug)]
 pub struct BranchRecord {
-    pub id: String,
+    pub id: Arc<String>,
```

Comment:
It looks like replacing `String` with `Arc<String>` breaks compilation since it does not implement the `Serialize` trait that we use to write records to csv.

Reply:
Oops I forgot to commit something